go/oasis-net-runner/cmd/root.go

```diff
@@ -173,8 +192,11 @@ func init() {
 
 	rootCmd.PersistentFlags().AddFlagSet(rootFlags)
 	rootCmd.PersistentFlags().AddFlagSet(env.Flags)
+	rootCmd.PersistentFlags().AddFlagSet(fixtures.DefaultFlags)
```

Comment:
Why are these persistent? Just because they are also needed by the dump-fixture command? I would rather add them to both commands (root and the dump-fixture subcommand).